src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private ColorSupplier colorSupplier = new ColorSupplier();
+    private Random random = new Random();
+    private final int indexRandom = random.nextInt(100);
```

Comment:
constants are static final fields that should have PROPER_NAMING_PATTERN (MAX_COUNT)
100 is magic number and it should be constant (not a random number)

```suggestion
    private static final int MAX_COUNT = 100;
```

Reply:
I know but mvn shows an error build. Ok I change it